Database/Updates/World/06-06-27-2017-generator-chains-testdata.sql

```diff
@@ -0,0 +1,122 @@
+USE `ace_world`;
+
+/* Create new generator link table */
+CREATE TABLE IF NOT EXISTS `ace_object_generator_link` (
+  `aceObjectId` int(10) unsigned NOT NULL,
+  `index` tinyint(3) unsigned NOT NULL,
+  `generatorId` int(10) unsigned NOT NULL,
```

Comment:
Yes the column refers to the wcid of the generator to spawn as the linked generator to spawn will get its position  - either absolute or relative (= random in that landblock) - from its top level generator.